app/models/invenio_rdm_record_converter.rb

```diff
@@ -92,7 +92,7 @@ def versions(gf)
     Sufia::Export::VersionGraphConverter.new(gf.content.versions).versions
   end
 
-  def invennio_pids(doi)
+  def invenio_pids(doi)
```

Comment:
I'm glad you did this. After @fenekku pointed it out I was sad.

Reply:
😆 same here, I reviewed it.